api/apps/api/src/migrations/api/1691980238543-AddCostSurdaceApiDb.ts

```diff
@@ -0,0 +1,31 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddCostSurdaceApiDb1691980238543 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+    CREATE TABLE cost_surfaces (
+      id uuid PRIMARY KEY DEFAULT gen_random_uuid(),
+      project_id uuid references projects(id),
+      name VARCHAR NOT NULL,
+      is_default BOOLEAN NOT NULL DEFAULT FALSE,
+      created_at timestamp without time zone NOT NULL DEFAULT now(),
+      created_by uuid NOT NULL REFERENCES users(id),
+      last_modified_at timestamp without time zone NOT NULL DEFAULT now(),
+      CHECK (name <> '')
+    );
```

Comment:
I suggest to also store here the `[min, max]` cost range of each cost surface.

it could be two float columns, or a 2-element float array (enforcing that cardinality is 2)

Reply:
Again the sort of stuff I am not sure about. Let's bring this up later, cheers!